.rpm_version

```diff
@@ -1 +1 @@
-1.12.18
+1.12.19
```

Comment:
This is going to end up overwriting the 1.12.19 rpms from the 1.2 branch.   We need to figure out what we are going to do with versions if we are going to have a 1.2.5 branch.  I think we need further discussion on this, but for now we should at least bump the minor version so we are not overwriting 1.2 builds.

Reply:
Increase the 1.2.5 version to 1.13.0 now.